openshift-client/src/main/java/io/fabric8/openshift/client/dsl/internal/DeploymentConfigOperationsImpl.java

```diff
@@ -115,4 +116,19 @@ public void run() {
       }
     }
   }
+
+  @Override
+  public DeploymentConfig scale(int count) {
+    return scale(count, false);
+  }
+
+  @Override
+  public DeploymentConfig scale(int count, boolean wait) {
+    DeploymentConfig res = cascading(false).edit().editSpec().withReplicas(count).endSpec().done();
+    if (wait) {
+      res = getMandatory();
```

Comment:
Sure, but as mentioned on #512 we can't wait for DeploymentConfig, as we don't have access to the replicas from the status.